executing/executing.py

```diff
@@ -509,9 +530,556 @@ def visit_ClassDef(self, node):
+            node = self.find_node(lasti)
+        except NotOneValueFound:
+            # LOAD_METHOD could load "".join for long "..."%(...) BinOps
+            # this can only be associated by using all positions
+            if self.opname(lasti) in (
+                "LOAD_METHOD",
+                "LOAD_ATTR",
+                "STORE_ATTR",
+                "DELETE_ATTR",
+            ):
+                # lineno and col_offset of LOAD_METHOD and *_ATTR instructions get set to the beginning of
+                # the attribute by the python compiler to improved error messages (PEP-657)
+                # we ignore here the start position and try to find the ast-node just by end position and expected node type
+                # This is save, because there can only be one attribute ending at a specific point in the source code.
+                node = self.find_node(
+                lasti,
+                match_positions=("end_col_offset", "end_lineno"),
+                typ=ast.Attribute,
+            )
+            elif self.opname(lasti) == "CALL":
+                # A CALL instruction can be a method call, in which case the lineno and col_offset gets changed by the compiler.
+                # Therefore we ignoring here this attributes and searchnig for a Call-node only by end_col_offset and end_lineno. 
+                # This is save, because there can only be one method ending at a specific point in the source code.
+                # One closing ) only belongs to one method.
+                node = self.find_node(
+                    lasti,
+                    match_positions=("end_col_offset", "end_lineno"),
+                    typ=ast.Call,
+                )
+            else:
+                raise
+
+        # report KnownIssues
+
+        instruction = self.instruction(lasti)
+
+        if instruction.opname in ("COMPARE_OP", "IS_OP", "CONTAINS_OP") and isinstance(
+            node, self.types_cmp_issue
+        ):
+            if isinstance(node, self.types_cmp_issue_fix):
+                # this is a workaround for https://github.com/python/cpython/issues/95921
+                # we can fix cases with only on comparison inside the test condition
+                #
+                # we can not fix cases like:
+                # if a<b<c and d<e<f: pass
+                # if (a<b<c)!=d!=e: pass
+                # because we don't know which comparison caused the problem
+
+
+                comparisons = [
+                    n
+                    for n in ast.walk(node.test)
+                    if isinstance(n, ast.Compare) and len(n.ops) > 1
+                ]
+
+                assert_(comparisons, "expected at least one comparison")
+
+                if len(comparisons) == 1:
+                    node = comparisons[0]
+                else:
+                    raise KnownIssue(
+                        "multiple chain comparison inside %s can not be fixed" % (node)
+                    )
```

Comment:
This is actually a lot simpler, but the `KnownIssue` type transports a meaning and is handled special in the test code (known issues are just ignored). It would not be possible to test a lot of source files if we would not ignore this kinds of issues.